rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel    = "1.90"
+components = ["clippy", "rust-src", "rustfmt"]
```

Comment:
My experience with this has usually been pretty bad. What inevitably happens is that I open a repo such as this; some random version of rust gets downloaded, and I don't get rust-analyzer support..

Reply:
For the record, @huitseeker did not want to specify the exact version - but @bitwalker and I were in favor of it :)

One of the arguments is that it enforces that all of us are building the codebase with the same version of Rust.

Reply:
@Mirko-von-Leipzig I know both Zed and VSCode allow configuring what rust-analyzer binary is used, both globally and project-local, should you need to. That said, I've never observed the presence of a `rust-toolchain.toml` with a specific toolchain breaking rust-analyzer support in my editor, and I use the default configuration in Zed for Rust.

Something to keep our eye on though for sure.

Reply:
If you're using a globally set version then the toolchain version isn't really helping or no? If we want to develop with the toolchain version then specifically RA should be running that version?

Or does it do something more special and accounts for the toolchain version even though RA is running on version whatever?